lte/gateway/python/magma/subscriberdb/client.py

```diff
@@ -77,32 +80,113 @@ def __init__(
         self._grpc_client_manager = grpc_client_manager
 
     async def _run(self) -> None:
-        subscribers = await self._get_subscribers()
-        if subscribers is not None:
-            self._process_subscribers(subscribers)
+        in_sync = await self._check_subscribers_in_sync()
+        if in_sync:
+            return
+
+        resync = await self._sync_subscribers()
+        if not resync:
+            return
+
```

Comment:
Let's remove all the sync_subs logic from the gw at this point, just cause it's not doing anything positive, at the cost of an extra round-trip per update